tensorflow_addons/image/transform_ops.py

```diff
@@ -35,10 +38,11 @@
 def transform(
     images: TensorLike,
     transforms: TensorLike,
-    interpolation: str = "NEAREST",
-    fill_mode: str = "CONSTANT",
+    interpolation: str = "nearest",
+    fill_mode: str = "constant",
```

Comment:
This will not break the compatibility as we always uppercase it when passing into raw ops.